example/rabbitmq/readme.md

```diff
@@ -0,0 +1,61 @@
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+# RabbitMQ-IoTDB Demo
+## Function
+```
+The example is to show how to send data from localhost to IoTDB through RabbitMQ.
+```
+## Usage
+### Version usage
+IoTDB: 0.10.0  
+RabbitMQ: 3.8.15
```

Comment:
Hi, could you plz let me know why using IoTDB 0.10.0 and RabbitMQ 3.8.15 here.
I found the RabbitMQ version you used right below is 5.12.0...

Reply:
> Hi, could you plz let me know why using IoTDB 0.10.0 and RabbitMQ 3.8.15 here.
> I found the RabbitMQ version you used right below is 5.12.0...

sorry, I forgot to modify the `IoTDB` version, It should be the newest version `0.12.0`.

But the `3.8.15` is RabbitMQ's server version, `5.12.0` is client jar's version? which is here?

Reply:
> But the `3.8.15` is RabbitMQ's server version, `5.12.0` is client jar's version? which is here?

I think it will be better if indicate the version clearly. 
Like
```
RabbitMQ server: 3.8.15
RabbitMQ client jar: 5.12.0
```
How do you think?